packages/open-next/src/core/routingHandler.ts

```diff
@@ -22,12 +22,17 @@ export interface MiddlewareOutputEvent {
   origin: Origin | false;
 }
 
+// Add the locale prefix to the regex so we correctly match the rawPath
+const optionalLocalePrefixRegex = !!RoutesManifest.locales.length
+  ? `^/(?:${RoutesManifest.locales.map((locale) => `${locale}/?`).join("|")})?`
+  : null;
```

Comment:
This make `pnpm run build` fail for me, in case people don't have i18n it should be the same so `^/`